README.md

```diff
@@ -12,18 +12,111 @@ Initial plan is to validate containers/images/dockerfiles against different ecos
 
 ![example](./docs/example.gif)
 
-## Technical details
 
-*Colin* will be available as a Python API, and will provide command line interface so you can easily use it locally.
+## Usage
 
-Each ecosystem will define a set of checks to validate the artifacts. Checks will have different severity level so that we can classify checks as required or optional.
 
-## Installation
+### Installing via `pip`
+
+We haven't released colin on PyPI yet, in the meantime, you can install it directly from git:
 
 ```bash
+$ pip3 install --user git+https://github.com/user-cont/colin.git
+```
+
+This is how you can use colin afterwards:
+
+```
+$ colin -h
+Usage: colin.py [OPTIONS] TARGET
+
+Options:
+  -c, --config TEXT  Configuration name.
```

Comment:
```
colin -h
Usage: colin [OPTIONS] TARGET

Options:
  -c, --config [rhel|fedora]  Select a predefined configuration.
  -f, --config-file FILENAME    Path to a file to use for validation (by
                                default they are placed in /usr/share/colin).
  --json FILENAME               File to save the output as json to.
  -s, --stat                    Print statistics instead of full results.
  -h, --help                    Show this message and exit.
```